Source/WEB_ROOT/admin/faculty/asset/editAsset_not_needed.html

```diff
@@ -0,0 +1,460 @@
+				<td class="formElement">
+					<input type="text" name="[u_assettable]Asset_Location" value="" />
+				</td>
+			</tr>
+			<tr>
+			    ~[tlist_sql;
+        	        Select 
+        	            Asset_Accessory_1_Name
+                    FROM U_assetsettings
+        	    ;]
+            	    ~(rename;t;if.blank.then=<td class="bold">Accessory 1</td>
+                    ;else=
+                    	<td class="bold" id=~(location)>~(rename)</td>
+                    )
+        	    [/tlist_sql] 
+				<td class="formElement">
+					<input type="text" name="[u_assettable]Asset_Accessory_1" value="" />
+				</td>
+			</tr>
+			<tr>
+			    ~[tlist_sql;
+        	        Select 
+        	            Asset_Accessory_2_Name
+                    FROM U_assetsettings
+        	    ;]
+            	    ~(rename;t;if.blank.then=<td class="bold">Accessory 2</td>
+                    ;else=
+                    	<td class="bold" id=~(location)>~(rename)</td>
+                    )
+        	    [/tlist_sql] 
+				<td class="formElement">
+					<input type="text" name="[u_assettable]Asset_Accessory_2" value="" />
+				</td>
+			</tr>
+			<tr>
+			    ~[tlist_sql;
+        	        Select 
+        	            Asset_Accessory_3_Name
+                    FROM U_assetsettings
+        	    ;]
+            	    ~(rename;t;if.blank.then=<td class="bold">Accessory 3</td>
+                    ;else=
+                    	<td class="bold" id=~(location)>~(rename)</td>
+                    )
+        	    [/tlist_sql] 
+				<td class="formElement">
+					<input type="text" name="[u_assettable]Asset_Accessory_3" value="" />
+				</td>
+			</tr>
+			<tr>
+				<td class="bold">Device Username</td>
+				<td class="formElement">
+					<input type="text" name="[u_assettable]DEVICE_USERNAME" value="" />
+				</td>
+			</tr>
+			<tr>
+				<td class="bold">Device Password</td>
+				<td class="formElement">
+					<input type="text" name="[u_assettable]DEVICE_PASSWORD" value="" /> <font size="1" color=red> NOT Encrypted in database</font>
+				</td>
+			</tr>
+			<tr>
+				<td class="bold">Notes</td>
+				<td class="formElement">
+					<textarea type="textarea" form ="assetform" name="[u_assettable]ASSET_NOTES" id="taid" rows="10" cols="35" wrap="soft"></textarea>
+				</td>
+			</tr>
+			<tr>
+				<td class="bold">CheckOut Date</td>
+				<td class="formElement">
+					<input type="text" name="[u_assettable]ASSET_CHECKOUTDATE" value="" />
+				</td>
+			</tr>
+			<tr>
+				<td class="bold">CheckIn Date</td>
+				<td class="formElement">
+					<input type="text" name="[u_assettable]ASSET_CHECKINDATE" value="" />
+				</td>
+			</tr>
+		</table>
+		<input type="hidden" id="isStudent_dcid" name="[u_assettable]IsStudent_dcid" value="" readonly /></input>
+		
+		<div class="button-row formElement">
+			<input type="hidden" name="ac" value="prim" />
+			<a href="" class="button">Cancel</a>
+			~[submitbutton]
+        </div>
+	</div> 
+</form>
+
+<script>

```

Comment:
Much like you've done earlier, throwing this in a javascript file has a couple of huge benefits. 1. Single Responsibility Principle (SRP) - the class does one thing, and only one thing well. Meaning this .html file's job is to determine the structure of the page. The .css has the job of determining the look of the page. 3. The javascript file has the job of determine how the page interacts.